### ISSUE TITLE
Problem on symfony 5.0

### ISSUE THREAD

**Issue by gbrle** [FriendsOfSymfony/FOSCKEditorBundle#205]
Hello,

i try install on symfony 5 but impossible !

How can fix ?

ERROR  : 

composer require friendsofsymfony/ckeditor-bundle
Using version ^2.1 for friendsofsymfony/ckeditor-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for friendsofsymfony/ckeditor-bundle ^2.1 -> satisfiable by friendsofsymfony/ckeditor-bundle[2.1.0].
    - friendsofsymfony/ckeditor-bundle 2.1.0 requires symfony/asset ^3.4 || ^4.0 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.

Thx you so much :)

**Comment by ChristopheFerreboeuf** [comment]
It just says that the bundle is not available yet for symfony 5.0 and it must be more than 3.4 or 4.0 for older versions of symfony. I face the same problem. There's a pull request, I'll try to check what is wrong with it.

**Comment by meniam** [comment]
> I'll try to check what is wrong with it.

Hello Christophe, any news?

**Comment by ChristopheFerreboeuf** [comment]
I did not have time to really check. I am quite busy at the moment.

**Comment by sergiu-popa** [comment]
In Symfony 5, command must return an int. So the return type was changed: https://github.com/FriendsOfSymfony/FOSCKEditorBundle/pull/201/commits/df99e286df247fe768a201cffff46ce1ae29cc80

I think because of this the tests fails:
```
1) FOS\CKEditorBundle\Tests\Installer\CKEditorInstallerTest::testInstall

Failed asserting that file "/home/travis/build/FriendsOfSymfony/FOSCKEditorBundle/tests/Installer/../../src/Resources/public/samples" does not exist.
```

**Comment by napestershine** [comment]
I can't use on SF 5

```
$  composer require friendsofsymfony/ckeditor-bundle
Using version ^2.1 for friendsofsymfony/ckeditor-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for friendsofsymfony/ckeditor-bundle ^2.1 -> satisfiable by friendsofsymfony/ckeditor-bundle[2.1.0].
    - friendsofsymfony/ckeditor-bundle 2.1.0 requires symfony/asset ^3.4 || ^4.0 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by Zausenec** [comment]
@napestershine 
Use ```composer req friendsofsymfony/ckeditor-bundle:2.x-dev``` until tagged release

**Comment by jmsche** [comment]
I think this issue can be closed as #201 was released some time ago
